### DISCUSSION TITLE
Postgres support

### DISCUSSION THREAD

**Discussion by jgkawell** [navidrome/navidrome#2820] [Ideas]
I like hosting all my services in Kubernetes and using sqlite for persistence means extra effort (for backups) and risk to my data versus connecting to a (pre-existing) postgres database hosted outside of navidrome. I would like the option to use postgres as the persistence layer instead of sqlite to support a more cloud native approach to hosting.

I actually have a fork with the changes to the persistence layer necessary for postgres support and will have that PR up shortly. Just putting this discussion here as a placeholder.

Note: while I was looking around for this being raised previously I found a few mentions of people wanting this feature as well: #685, #2258

**Comment by elijahgronda** [comment]
Hey! Been a while since you posted this but thought I’d ask, did you ever make any progress on this? Surprised to see you’re the only person to open this discussion.
